src/foam/nanos/crunch/InvalidateDependenciesIfPrerequisiteInvalidated.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'InvalidateDependenciesIfPrerequisiteInvalidated',
```

Comment:
perhaps a name to change to 
CascadeInvalidate
but I now see that many of other crunch rules have very verbose names.

Reply:
that sounds good, thanks, done @jlhughes 
https://github.com/foam-framework/foam2/pull/3577